[docs] CDC: Update content for CDC streams and transactional ordering (#25112)
yugabyte/yugabyte-db: docs/content/preview/admin/yb-admin.md

CDC Stream ID: d540f5e4890c4d3b812933cbfd703ed3
```

> {{< note title="Note" >}}

Reviewer: ```suggestion
{{< note title="IMPLICIT checkpointing is deprecated" >}}
```

> It is recommended that you create a stream ID with `EXPLICIT` checkpointing mode only, which is also the default checkpointing mode now. The `IMPLICIT` checkpointing mode will be completely removed in future releases.

Reviewer: ```suggestion
It is recommended that you create streams in EXPLICIT checkpointing mode only (the default). IMPLICIT checkpointing mode will be completely removed in future releases.
```

{{ < /note > }}

> 

Reviewer 2: `Note` is not displaying correctly. Please look at the note implementation in other places to get an idea.
![image](https://github.com/user-attachments/assets/8eaea6af-2dec-4548-b4ab-a4d7a0132853)

Reviewer 2: This is also a depreciation notice, right? then should we use a warning instead of a note here? You can check the example of a warning note in this doc https://deploy-preview-25112--infallible-bardeen-164bc9.netlify.app/preview/explore/colocation/#databases

Reviewer 3: This is not a deprecation notice yet, we do support both modes but we are recommending using only `EXPLICIT`

Reviewer 2: Okay!
